client/app/react-components/ChartSeriesEditor.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```

Comment:
In general remove inline styles here. I am not sure why the resizable component is a table though it might be intentional. Styling tables is quite tricky especially cross-browser.